app/routes/goal_routes.py

```diff
@@ -1 +1,93 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, Response
+from ..routes.utilities import create_model
+from ..models.goal import Goal
+from ..db import db
+from ..routes.utilities import validate_model
+from ..models.task import Task
```

Comment:
[PEP-8 Style Guide](https://peps.python.org/pep-0008/#imports) declares that it is okay to have imports from the same module on the same line.
```suggestion
from ..routes.utilities import create_model, validate_model
from ..models.goal import Goal
from ..db import db
from ..models.task import Task
```